commands/deploy.go

```diff
@@ -270,7 +298,18 @@ Error: %s`, fprocessErr.Error())
 			gateway = getGatewayURL(gateway, defaultGateway, gateway, os.Getenv(openFaaSURLEnvironment))
 			registryAuth = getRegistryAuth(&dockerConfig, image)
 		}
-		statusCode, err := deployImage(image, fprocess, functionName, registryAuth, deployFlags, tlsInsecure)
+		// default to a readable filesystem until we get more input about the expected behavior
+		// and if we want to add another flag for this case
+		defaultReadOnlyRfs := false
```

Comment:
I don't mind renaming this variable, but if we want to be consistent with core go, this should be `defaultReadOnlyRFS` per https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
👍 This was actually a new variable. I can make this change, though.